src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandler.java

```diff
@@ -0,0 +1,103 @@
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NotSuitable_SDO;
+
+@Service
+@RequiredArgsConstructor
+public class NotSuitableSDOCallbackHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(NotSuitable_SDO);
+    public static final String NotSuitableSDO_CONFIRMATION_BODY = "<br />If a Judge has submitted this information, "
+        + "a notification will be sent to the listing officer to look at this case offline."
+        + "%n%nIf a legal adviser has submitted this information a notification will be sent to a judge for review.";
+
+    private final ObjectMapper objectMapper;
+
+    private final Time time;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return new ImmutableMap.Builder<String, Callback>()
+            .put(callbackKey(ABOUT_TO_START), this::addUnsuitableSDODate)
+            .put(callbackKey(ABOUT_TO_SUBMIT), this::submitNotSuitableSDO)
+            .put(callbackKey(SUBMITTED), this::buildConfirmation)
+            .build();
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private CallbackResponse submitNotSuitableSDO(CallbackParams callbackParams) {
+        CaseData.CaseDataBuilder dataBuilder = getSharedData(callbackParams);
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(dataBuilder.build().toMap(objectMapper))
+            .build();
+    }
+
+    private CallbackResponse addUnsuitableSDODate(CallbackParams callbackParams) {
+        CaseData.CaseDataBuilder dataBuilder = callbackParams.getCaseData().toBuilder();
+
+        dataBuilder.unsuitableSDODate(time.now());
+
+        return AboutToStartOrSubmitCallbackResponse.builder()
+            .data(dataBuilder.build().toMap(objectMapper))
+            .build();
+    }
+
+    private CaseData.CaseDataBuilder getSharedData(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder dataBuilder = caseData.toBuilder();
+
+        dataBuilder.businessProcess(BusinessProcess.ready(NotSuitable_SDO));
+
+        return dataBuilder;
+    }
+
+    private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+
+        return SubmittedCallbackResponse.builder()
+            .confirmationHeader(getHeader(caseData))
+            .confirmationBody(getBody(caseData))
+            .build();
+    }
+
+    private String getHeader(CaseData caseData) {
+        return format("# Your request was accepted%n## Case has now moved offline");
+    }
+
+    protected CallbackResponse emptyCallbackResponse(CallbackParams callbackParams) {
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    private String getBody(CaseData caseData) {
+        return format(NotSuitableSDO_CONFIRMATION_BODY);
```

Comment:
Naming is not very consistent

Reply:
This class was merged in from the CIV-3328 branch. I only made the method 'addUnsuitableSDODate'.